src/Illuminate/Mail/Mailable.php

```diff
@@ -167,11 +167,11 @@ class Mailable implements MailableContract, Renderable
      * Send the message using the given mailer.
      *
      * @param  \Illuminate\Contracts\Mail\Factory|\Illuminate\Contracts\Mail\Mailer  $mailer
-     * @return void
+     * @return \Symfony\Component\Mailer\SentMessage|null
```

Comment:
Thoughts on decorating this so that we can add more convenience features we may need in the future without a breaking change?

Reply:
Yeah, I will send a PR for that.

@driesvints mentioned in the PR description
`We could wrap Symfony\Component\Mailer\SentMessage to add some nice API methods like retrieving recipients, etc`

Reply:
PR: https://github.com/laravel/framework/pull/38803